root/defaults/default

```diff
@@ -1,28 +1,39 @@
 server {
-	listen 80 default_server;
-	root /var/www/localhost/lychee;
-	index index.html index.htm index.php;
-
-	server_name _;
-	client_max_body_size 0;
-
-	location / {
-		try_files $uri $uri/ /index.html /index.php?$args =404;
-		}
-
-	 location ~* \.(?:ico|css|js|gif|jpe?g|png)$ {
-                expires max;
-                add_header Pragma public;
-                add_header Cache-Control "public, must-revalidate, proxy-revalidate";
-        }
-
-	location ~ \.php$ {
-		fastcgi_split_path_info ^(.+\.php)(/.+)$;
-		fastcgi_pass 127.0.0.1:9000;
-		fastcgi_index index.php;
-		fastcgi_read_timeout 1800;
-		fastcgi_send_timeout 1800;
-		include /etc/nginx/fastcgi_params;
-
-	}
+    listen 80 default_server;
+
+    listen 443 ssl;
+
+    root /app/lychee/public;
+    index index.php index.html index.htm;
+
+    server_name _;
+
+    ssl_certificate /config/keys/cert.crt;
+    ssl_certificate_key /config/keys/cert.key;
+
+    client_max_body_size 0;
+
+    error_page 599 = @noauth;
+
+    location @noauth {
```

Comment:
is this needed? I don't see an http auth setting

Reply:
nope, removed.